ui-grid/ui-grid.d.ts

```diff
@@ -927,6 +927,12 @@ declare namespace uiGrid {
          */
         notifyDataChange(type: string): void;
         /**
+         * Refresh the rendered grid on screen.
+         *
+         * @param {boolean} [rowsAltered] Optional flag for refreshing when the number of rows has changed.
+         */
+        refresh(rowsAltered?: boolean): ng.IPromise<any>;
```

Comment:
I'm not seeing any reference to the `rowsAltered` parameter in the documentation.  This is what I see:

> `refresh()`
>
>Refresh the rendered grid on screen. The refresh method re-runs both the columnProcessors and the rowProcessors, as well as calling refreshCanvas to update all the grid sizing. In general you should prefer to use queueGridRefresh instead, which is basically a debounced version of refresh.
>
>If you only want to resize the grid, not regenerate all the rows and columns, you should consider directly calling refreshCanvas instead.

Could you provide a reference to the documentation supporting the parameter, and if the documentation is wrong, would you mind submitting a change to ui-grid's documentation repo and reference this pull request?